target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -130,12 +130,14 @@ case "$FIRMWARE" in
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
 		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
 		;;
-	comfast,cf-e560ac)
+	comfast,cf-e560ac|\
+	comfast,cf-wr752ac-v1)
 		caldata_extract "art" 0x5000 0x2f20
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) +2)
 		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
 			/lib/firmware/ath10k/QCA9888/hw2.0/board.bin
-		rm /lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin
+		board2file="/lib/firmware/ath10k/QCA9888/hw2.0/board-2.bin"
+		[ -e $board2file ] && rm $board2file
```

Comment:
Why do you think this is required?

Reply:
Just to avoid error log entries in case there is no such file. It's just cleaner and doesn't hurt.

Reply:
I can read that, but why do you expect that the existence of the file is random? If there is no file there, you wouldn't need the rm at all. So, please check whether you need to remove the file or not.
What you've done here is actually a workaround for not having to look ...

Reply:
OK, if you don't want it, I take it out.